Symfony/CS/Console/Application.php

```diff
@@ -48,4 +51,31 @@ public function getLongVersion()
 
         return $version;
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function doRun(InputInterface $input, OutputInterface $output)
+    {
+        $stdErr = $output instanceof ConsoleOutputInterface
+            ? $output->getErrorOutput()
+            : ($input->hasParameterOption('--format', true) && 'txt' !== $input->getParameterOption('--format', null, true) ? null : $output)
+        ;
+
+        if (null !== $stdErr) {
+            $warningsDetector = new WarningsDetector();
+            $warningsDetector->detectOldVendor();
+            $warningsDetector->detectOldMajor();
+
+            if (FixCommand::COMMAND_NAME === $this->getCommandName($input)) {
+                $warningsDetector->detectXdebug();
+            }
+
+            foreach ($warningsDetector->getWarnings() as $warning) {
+                $stdErr->writeln(sprintf($stdErr->isDecorated() ? '<bg=yellow;fg=black;>%s</>' : '%s', $warning));
```

Comment:
`stdErr` is an instance of `OutputInterface` and we don't check the formatter, so I think we cannot be sure escaping is done here always? If we assume we can rely on this on other places than lets remove the check here as well in a new PR, otherwise lets leave as is ;)